frontend/public/components/utils/details-page.tsx

```diff
@@ -91,20 +97,20 @@ export const ResourceSummary: React.SFC<ResourceSummaryProps> = ({
         <LabelList kind={reference} labels={metadata.labels} />
       </DetailsItem>
       {showPodSelector && (
-        <DetailsItem label="Pod Selector" obj={resource} path={podSelector}>
+        <DetailsItem label={t('details-page~Pod Selector')} obj={resource} path={podSelector}>
           <Selector
             selector={_.get(resource, podSelector)}
             namespace={_.get(resource, 'metadata.namespace')}
           />
         </DetailsItem>
       )}
       {showNodeSelector && (
-        <DetailsItem label="Node Selector" obj={resource} path={nodeSelector}>
-          <Selector kind="Node" selector={_.get(resource, nodeSelector)} />
+        <DetailsItem label={t('details-page~Node Selector')} obj={resource} path={nodeSelector}>
```

Comment:
The same case change will need to be made to other properties in this file.

```suggestion
        <DetailsItem label={t('details-page~Node selector')} obj={resource} path={nodeSelector}>
```